posts/inside-rust/2023-09-22-project-director-nominees.md

```diff
@@ -0,0 +1,32 @@
+# Announcing the Project Director Nominees
+
+Over the past couple weeks, the Rust Leadership Council has been soliciting nominees from the whole Rust project for the role of Project Director.
+Now that the nomination period is closed, we are happy to announce the list of nominees.
+As described in our [last post], we will elect from these people three new Project Directors.
+
+The nominees are:
+
+- Jakob Degen
+- Amanieu
+- Felix Klock
+- lcnr
+- scottmcm
+- Michael Goulet
+- Santiago Pastorino
+- Jubilee
+- Mara Bos
+
+These were all spoken of highly by their nominating teams.
+While several other candidates were suggested, not everyone who was nominated felt like they were in a place where they were able or wanted to accept the role if they were elected.
+
+Now we'd again like to ask the members of the Rust Project for their feedback.
+Please let us know if there's anything we should consider going into the election process.

```

Comment:
Let us know - can we be more specific? For example, email / Zulip / specific people contacted... I think in particular this may be relevant for Mara as representative and nominee, which could be a conflict of interest for that set of teams (at least potentially).

Reply:
I tried to rephrase this to be a little more specific. We did have a link to the council Zulip and email address down below, but I can move it up higher if that would make it more obvious.

I also added a paragraph about Mara's conflict of interest and how we will address that.